diff --git a/lib/types-CNjZgO4h.js b/lib/types-CNjZgO4h.js
index 6da3e5c..2164c3b 100644
--- a/lib/types-CNjZgO4h.js
+++ b/lib/types-CNjZgO4h.js
@@ -853,7 +853,8 @@ function spawnSandboxed(api, token, options) {
 	const startupInfo = allocStartupInfo();
 	encodeStartupInfo(startupInfo, {
 		cb: 104,
-		dwFlags: 256,
+		dwFlags: 257,
+		wShowWindow: 0,
 		hStdInput: stdIn.read,
 		hStdOutput: stdOut.write,
 		hStdError: stdErr.write
@@ -990,7 +991,8 @@ function spawnSandboxedInherited(api, token, options) {
 	const startupInfo = allocStartupInfo();
 	encodeStartupInfo(startupInfo, {
 		cb: 104,
-		dwFlags: 256,
+		dwFlags: 257,
+		wShowWindow: 0,
 		hStdInput: stdIn,
 		hStdOutput: stdOut,
 		hStdError: stdErr
